tests/integration/test_sns.py

```diff
@@ -786,6 +786,38 @@ def test_publish_sqs_from_sns(self):
         )
         self.assertEqual(len(response['Messages']), 1)
 
+    def add_xray_header(self, request, **kwargs):
+        request.headers['X-Amzn-Trace-Id'] = \
+            'Root=1-3152b799-8954dae64eda91bc9a23a7e8;Parent=7fa8c0f79203be72;Sampled=1'
+
+    def test_publish_sqs_from_sns_with_xray_propagation(self):
+
```

Comment:
Also, I think we should skip this test for moto backend:
```
from localstack.services.install import SQS_BACKEND_IMPL
...
if SQS_BACKEND_IMPL != 'elasticmq':
    return
```

Reply:
Excellent suggestions, thanks a lot. Just made the changes, will see how it goes.